code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -574,6 +578,8 @@
 	var/moved = target.Move(shove_to, shove_dir)
 	if(!moved) //they got pushed into a dense object
 		add_attack_logs(user, target, "Disarmed into a dense object", ATKLOG_ALL)
+		target.visible_message("<span class='warning'>[target] gets slammed into the wall!</span>", \
+								"<span class='warning'>You get slammed into the wall!</span>")
```

Comment:
we don't actually know what the `object` is in this proc. as it mostly relies on the default byond `Move()` proc to work. so without doing a loop on the turf to find a dense object I cannot do that.

Reply:
updated to be more general

Reply:
Hmm, alright. I still do think it'd be nice to get a bit more feedback, even if you do have to loop through the turf (though you could probably just optimize by checking if it's a wall, which I bet it will be 90% of the time).